app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        parameter, old_file, new_file = command.split()
```

Comment:
Please, don't split `command` twice. Create a variable for storing the result `command.split()` and use it on demand.